specs/eventing/test-plan/control-plane/broker-lifecycle/broker.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: eventing.knative.dev/v1
+kind: broker
+metadata:
+ name: conformance-broker
+ namespace: default
```

Comment:
conformance tests should not assume to be deployed in the default namespace. For instance IBM Code Engine does not have the `default` namespace.

Reply:
@lionelvillard good point

Reply:
@lionelvillard fixed in all files